druid/examples/readme.md

```diff
@@ -55,12 +55,31 @@ cargo run --example hello
 This shows some of the basics of druid. If you need a start of how to build an application with a text-box and some labels this is where to start.
 
 ## Hello_web
+Not working, browser is blank after going to http://localhost:8000?
+
+For more info and prerequistes see [druid/examples/hello_web/README.md](druid/examples/hello_web/README.md).
 ```
 cd druid/examples/hello_web
 wasm-pack build --out-dir pkg --out-name hello_web
+http
```

Comment:
I think we should not have this in the examples. How the user serves and loads this example is up to them not us.

Reply:
Done